### PR TITLE
[EDX-487] - Spacing H2 & LHS Top fixes

### PR DESCRIPTION
**Pull request by aralovelace**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

A PR description indicating the purpose of the PR.

* [EDX-487](https://ably.atlassian.net/browse/EDX-487?atlOrigin=eyJpIjoiNGYxNDdjYzM0MDJkNGQzNDg3YzBhOTI1ZjY4NDE3YWMiLCJwIjoiaiJ9)

## Review

Instructions on how to review the PR. 

* all pages

### CODE REVIEW

File: src/components/blocks/software/__snapshots__/Pre.test.tsx.snap [hunk 1]
```diff
@@ -20,7 +20,7 @@ exports[`<Pre /> should successfully render Code elements with language 1`] = `
     class="bg-cool-black text-white p-0 rounded-lg relative"
   >
     <menu
-      class="flex overflow-visible m-0 pl-0 scrollX justify-end md:justify-start h-48 mr-48"
+      class="flex overflow-visible m-0 pl-0 scrollX justify-end md:justify-start h-48 mr-48 navStyles"
```
[the remark this answers is not in the document]
Comment (aralovelace) [reply src/components/blocks/software/__snapshots__/Pre.test.tsx.snap:23]: @snikidev  I dont know it just updated when I run the test

File: src/components/Sidebar/SidebarLinkMenu.tsx [hunk 2]
```diff
@@ -56,8 +56,11 @@ export const SidebarLinkMenu = ({
             ? EXPAND_MENU.EXPANDED
             : expandMenu;
 
+        console.log('Safe Window: ' + safeWindow.location.pathname + ' highlightedMenuId: ' + highlightedMenuId);
```
[the remark this answers is not in the document]
Comment (aralovelace) [reply src/components/Sidebar/SidebarLinkMenu.tsx:59]: @MymmiJ  oh ya sorry I was debugging the staging issue will work in a new repo for this one

File: src/components/blocks/software/__snapshots__/Pre.test.tsx.snap (hunk 1 above)
[the remark this answers is not in the document]
Comment (aralovelace) [reply src/components/blocks/software/__snapshots__/Pre.test.tsx.snap:23]: @MymmiJ  yes I reckon this is from @snikidev  last changes cos we didn't run a test that time  when we merged hahaha

File: src/components/Sidebar/SidebarLinkMenu.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (aralovelace) [reply src/components/Sidebar/SidebarLinkMenu.tsx]: this is now removed

File: src/components/blocks/software/__snapshots__/Pre.test.tsx.snap (hunk 1 above)
[the remark this answers is not in the document]
Comment (DuncanEdwards) [reply src/components/blocks/software/__snapshots__/Pre.test.tsx.snap:23]: It is also the wrong casing.

[the remark this answers is not in the document]
Comment (snikidev) [reply src/components/blocks/software/__snapshots__/Pre.test.tsx.snap:23]: @DuncanEdwards do you mean it should be kebab case? if it's coming from module.css then it's fine. 🙃

**Comment by snikidev** [comment]
reopened in #1744

**Pull request closed without merging**